draft-jhoyla-req-mtls-flag.md

```diff
@@ -32,48 +32,99 @@ informative:
 --- abstract
 
 Normally in TLS there is no way for the client to signal to the server that it
-has been configured with a certificate suitable for mTLS. This document defines
-a TLS Flag {{!I-D.ietf-tls-tlsflags}} that enables clients to provide this hint.
+supports client authentication and will handle a `CertificateRequest`
+gracefully. This document defines a TLS Flag that enables clients to provide
+this hint.
 
 
 --- middle
 
 # Introduction
 
-This document specifies a TLS Flag that indicates to the server that the client
-supports mTLS. Sometimes a server does not want to negotiate mTLS with every
-client, but might wish to authenticate a subset of them. In TLS 1.3 this may be
-done with post-handshake auth, however this adds an extra round-trip, and
-requires negotiation at the application layer. A client sending the request
-mTLS flag in the ClientHello allows the server to request authentication during
-the initial handshake only when it receives a hint the client supports it. This
-enables a number of use cases, for example allowing bots to authenticate
-themselves when mixed in with general traffic.
+This document specifies a TLS Flag {{!I-D.ietf-tls-tlsflags}} that allows a
+client to attempt to negotiate mutually authenticated TLS (in at least some
+cases). Sometimes a server does not want to authenticate every client, but
```

Comment:
Along the lines of avoiding the term "request mutual TLS", we might rewrite this as
```suggestion
client to prompt the server to request a certificate during the handshake. Sometimes a server does not want to authenticate every client, but
```